documentation/1.10/content/quickstart/setup.md

```diff
@@ -5,6 +5,13 @@ draft: false
 weight: 2
 ---
 
-- Download and unzip the WIT release ZIP file to a desired location.
-- OPTIONALLY: You may build the project (`mvn clean package`) to create the ZIP installer in `./imagetool/target`.
+- Download the WIT release ZIP file to a desired location.
+  - The latest ZIP file can be found on the project [releases](https://github.com/oracle/weblogic-image-tool/releases) page.
+  - Alternatively, you can download the ZIP file with cURL.
+    ```shell
+    curl -m 120 -fL https://github.com/oracle/weblogic-image-tool/releases/latest/download/imagetool.zip -o ./imagetool.zip
+    ```
```

Comment:
1) "the ZIP file" --> the latest ZIP file
2) it'd be helpful IMO to also teach how to get a specific version (assuming that is straight forward) 
3) would it help if the non-quickstart samples could point to quick start?

Reply:
4. the sample assumes image tool bin is in the PATH, but has no instructions for setting same

Reply:
> 4. the sample assumes image tool bin is in the PATH, but has no instructions for setting same

I think we can assume that the user knows how to add a directory to the PATH.